frontend/Movie_Pigeon/app/src/main/java/org/example/team_pigeon/movie_pigeon/SearchPageFragment.java

```diff
@@ -1,63 +1,259 @@
+        movieListView.setOnScrollListener(new PauseOnScrollListener(ImageLoader.getInstance(), true, true, new AbsListView.OnScrollListener() {
+            @Override
+            public void onScrollStateChanged(AbsListView view, int scrollState) {
+            }
+
             @Override
-            public void onCheckedChanged(RadioGroup group, int checkedId) {
-                switch (checkedId) {
-                    case R.id.button_title:
-                        etSearch.setHint("Search movies by title...");
-                        break;
-                    case R.id.button_actor:
-                        etSearch.setHint("Search movies by actor...");
-                        break;
-                    case R.id.button_country:
-                        etSearch.setHint("Search movies by country...");
-                        break;
+            public void onScroll(AbsListView view, int firstVisibleItem, int visibleItemCount, int totalItemCount) {
+                int movieCountThreshold = nowShowingCount==0? 20 : 21+nowShowingCount;
+                if (view.getLastVisiblePosition() == totalItemCount-1 && movieListView.getCount() >= movieCountThreshold && !isLoading && !noMoreResult) {
+                    isLoading = true;
+                    searchMoreTask = new SearchMoreTask();
+                    searchMoreTask.execute();
                 }
             }
-        });
+        }));
         return view;
     }
 
+    private void bindView(View view) {
+        movieListView = (ListView) view.findViewById(R.id.list_movies);
+        toolbar = (Toolbar) getActivity().findViewById(R.id.toolbar_display_page);
+    }
+
+    @Override
+    public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+        boolean needSkip = !(nowShowingCount==0);
+        if(needSkip && position == nowShowingCount+1) {}
+        else{
+            android.app.FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
+            MoviePageFragment moviePageFragment = new MoviePageFragment();
+            Bundle bundle = new Bundle();
+            bundle.putSerializable("movie", (Serializable) parent.getAdapter().getItem(position));
+            bundle.putInt("position", position);
+            bundle.putString("type", "search");
+            toolbar.setTitle(((Movie) parent.getAdapter().getItem(position)).getTitle());
+            toolbar.setSubtitle(null);
+            moviePageFragment.setArguments(bundle);
+            fragmentTransaction.replace(R.id.fl_content, moviePageFragment);
+            fragmentTransaction.addToBackStack(null);
+            fragmentTransaction.commit();
+        }
+    }
+
+    @Override
+    public void onDestroy() {
+        super.onDestroy();
+        EventBus.getDefault().unregister(this);
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+    }
+
+    public void onPause() {
+        super.onPause();
+    }
+
```

Comment:
onResume and onPause override are not needed as nothing is changed inside the respective methods